src/04-date-tasks.js

```diff
@@ -73,8 +82,31 @@ function isLeapYear(/* date */) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,10,0,0,250)     => "00:00:00.250"
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
```

Comment:
the output looks similar to output of "Date.prototype.toISOString()"
![image](https://user-images.githubusercontent.com/22891941/200375654-986a37cf-2621-4943-8bf0-219b9837135d.png)